account/guides/setupNetworkBridge.md

```diff
@@ -116,17 +124,17 @@ Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
 ## Create Call Using the Network Bridge {#create-call}
 To create a call using the network bridge format:
 
-* `TO` field like `sip:{Desired_to_number}@custxx.auth.bandwidth.com:{Port}`
+* `TO` field like `sip:{Desired_to_number}@<randomAccountHex>.auth.bandwidth.com:{Port}`
 * `SipAuthUsername` as the username created above like `sipauthtest`
 * `SipAuthPassword` as the password used to create the MD5 hash like `password`
 
 ##### Format the `TO` Field to use the Network Bridge
-`sip:+17778889999@custxx.auth.bandwidth.com:5006`
+`sip:+17778889999@<randomAccountHex>.auth.bandwidth.com:5006`
 
 ##### Curl Request to Create the Call
 ```bash
 curl 'https://api.twilio.com/2010-04-01/Accounts/{AccountId}/Calls.json' -X POST \
---data-urlencode 'To=sip:+17778889999@custxx.auth.bandwidth.com:5006' \
+--data-urlencode 'To=sip:+17778889999@<randomAccountHex>.auth.bandwidth.com:5006' \
```

Comment:
Realms should be `realmname.<randomAccountHex>.auth.bandwidth.com` here

Reply:
done